stellar/its.js

```diff
@@ -61,6 +227,51 @@ if (require.main === module) {
             mainProcessor(removeTrustedChain, chainName, options);
         });
 
+    program
+        .command('deploy-interchain-token <symbol> <name> <decimals> <salt> <initialSupply> ')
+        .description('deploy interchain token')
+        .action((symbol, name, decimal, salt, initialSupply, options) => {
+            mainProcessor(deployInterchainToken, [symbol, name, decimal, salt, initialSupply], options);
+        });
+
+    program
+        .command('deploy-remote-interchain-token <salt> <destinationChain> <gasTokenAddress> <gasFeeAmount>')
```

Comment:
Think more about simplifying the UX here. gasToken will be XLM in normal usage, so use that address by default. Default gas amount to `0` for convenience on devnets.